libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -468,9 +468,11 @@ private void unbindTelemetryService() {
   private void stopTelemetryService() {
     TelemetryLocationEnabler.LocationState telemetryLocationState = telemetryLocationEnabler
```

Comment:
shouldn't null check be at this line? as well as should we escape early? 
`if (telemetryService == null) {
     /* log error */ 
     return;
  }
`